### PR TITLE
Google font

### PR DESCRIPTION
**Pull request by masterqwerty**
This adds support for google's font Domine. I put the font everywhere where the Sentinel font was.

### CODE REVIEW

**Comment by masterqwerty** [comment]
Hold on, I just realized that I missed a few things regarding font sizes.

**Comment by masterqwerty** [comment]
Here. There was only really one place that the font size needed to be adjusted. Everywhere else seemed to be just fine.

File: byu-calendar-row/byu-calendar-row.scss [hunk 1]
```diff
@@ -208,5 +208,5 @@ p, div, a {
 }
 
 h1, h2, h3, h4 {
-	font-family: "Sentinel A", "Sentinel B";
+	font-family: Domine, "Sentinel A", "Sentinel B";
```
Comment (katrialesser) [inline byu-calendar-row/byu-calendar-row.scss]: You also need to include a basic fallback - 
font-family: 'Domine', serif; is the example google gives. Let's add that after Sentinel lines.

File: byu-calendar/byu-calendar-common.scss [hunk 2]
```diff
@@ -24,9 +24,9 @@
 }
 .fullpage-date-weekday {
     text-transform: uppercase;
-    font-family: "Sentinel A", "Sentinel B", "Vitesse A", "Vitesse B", sans-serif;
+    font-family: Domine, "Sentinel A", "Sentinel B", "Vitesse A", "Vitesse B", sans-serif;
```
Comment (katrialesser) [inline byu-calendar/byu-calendar-common.scss]: it shouldn't have sans-serif, it should be serif
font-family: 'Domine', serif;

File: byu-calendar/byu-calendar-common.scss [hunk 3]
```diff
@@ -24,9 +24,9 @@
...
 }
 .fullpage-date-text {
     padding-left: 10px;
-    font-family: "Sentinel A", "Sentinel B", "Vitesse A", "Vitesse B", sans-serif;
+    font-family: Domine, "Sentinel A", "Sentinel B", "Vitesse A", "Vitesse B", sans-serif;
```
Comment (katrialesser) [inline byu-calendar/byu-calendar-common.scss]: it shouldn't have sans-serif, it should be serif
font-family: 'Domine', serif;

**Pull request merged**